docs/advanced_usage.rst

```diff
@@ -0,0 +1,333 @@
+is a small example showcasing this::
+
+    from ddtrace import tracer
+
+    tracer.configure(hostname=<YOUR_HOST>, port=<YOUR_PORT>)
+
+By default, these will be set to localhost and 8126 respectively.
+
+Distributed Tracing
+-------------------
+
+To trace requests across hosts, the spans on the secondary hosts must be linked together by setting `trace_id`, `parent_id` and `sampling_priority`.
+
+- On the server side, it means to read propagated attributes and set them to the active tracing context.
+- On the client side, it means to propagate the attributes, commonly as a header/metadata.
+
+`ddtrace` already provides default propagators but you can also implement your own.
+
+Web Frameworks
+^^^^^^^^^^^^^^
+
+Some web framework integrations support the distributed tracing out of the box, you just have to enable it.
+For that, refer to the configuration of the given integration.
+Supported web frameworks:
+
+- Django
+- Flask
+- Tornado
+
+For web servers not supported, you can extract the HTTP context from the headers using the `HTTPPropagator`.
+
+.. autoclass:: ddtrace.propagation.http.HTTPPropagator
+    :members: extract
+
+HTTP Client
+^^^^^^^^^^^
+
+When calling a remote HTTP server part of the distributed trace, you have to propagate the HTTP headers.
+This is not done automatically to prevent your system from leaking tracing information to external services.
+
+.. autoclass:: ddtrace.propagation.http.HTTPPropagator
+    :members: inject
+
+Custom
+^^^^^^
+
+You can manually propagate your tracing context over your RPC protocol. Here is an example assuming that you have `rpc.call`
+function that call a `method` and propagate a `rpc_metadata` dictionary over the wire::
+
+
+    # Implement your own context propagator
+    MyRPCPropagator(object):
+        def inject(self, span_context, rpc_metadata):
+            rpc_metadata.update({
+                'trace_id': span_context.trace_id,
+                'span_id': span_context.span_id,
+                'sampling_priority': span_context.sampling_priority,
+            })
+
+        def extract(self, rpc_metadata):
+            return Context(
+                trace_id=rpc_metadata['trace_id'],
+                span_id=rpc_metadata['span_id'],
+                sampling_priority=rpc_metadata['sampling_priority'],
+            )
+
+    # On the parent side
+    def parent_rpc_call():
+        with tracer.trace("parent_span") as span:
+            rpc_metadata = {}
+            propagator = MyRPCPropagator()
+            propagator.inject(span.context, rpc_metadata)
+            method = "<my rpc method>"
+            rpc.call(method, metadata)
+
+    # On the child side
+    def child_rpc_call(method, rpc_metadata):
+        propagator = MyRPCPropagator()
+        context = propagator.extract(rpc_metadata)
+        tracer.context_provider.activate(context)
+
+        with tracer.trace("child_span") as span:
+            span.set_meta('my_rpc_method', method)
+
+
+Sampling
```

Comment:
This information should be kept outside of this documentation and should instead be centralized in the official APM docs.

All we should keep here is _how_ to configure sampling in the Python client.